command/repo.go

```diff
@@ -2,19 +2,30 @@ package command
 
 import (
 	"fmt"
+	"net/url"
 	"os"
 	"strings"
+	"time"
 
+	"github.com/cli/cli/api"
 	"github.com/cli/cli/git"
 	"github.com/cli/cli/internal/ghrepo"
 	"github.com/cli/cli/utils"
+
+	"github.com/AlecAivazis/survey/v2"
 	"github.com/spf13/cobra"
 )
 
 func init() {
 	RootCmd.AddCommand(repoCmd)
 	repoCmd.AddCommand(repoCloneCmd)
 	repoCmd.AddCommand(repoViewCmd)
+	repoCmd.AddCommand(repoForkCmd)
+
+	repoForkCmd.Flags().String("clone", "prompt", "Clone fork. Pass false to not clone fork.")
+	repoForkCmd.Flags().String("remote", "prompt", "Add remote for fork. Pass false to not add remote.")
```

Comment:
Nit: could we generally avoid punctuation and multiple sentences within flags docs? This is for the same reason why we would avoid them when raising error messages in Go: so that the end formatter (in this case, `gh help`) has more control over the final output.

I'm also thinking that it would be useful to list possible flag values like we do elsewhere. I wonder if this would do the trick:
```go
repoForkCmd.Flags().String("clone", "prompt", "Clone the new fork: {true|false|prompt}")
repoForkCmd.Flags().String("remote", "prompt", "Add a git remote for the fork: {true|false|prompt}")
```

The help formatter would then append the default value in the final output:
```
Flags:
  --clone string    Clone the new fork: {true|false|prompt} (default "prompt")
  --remote string   Add a git remote for the fork: {true|false|prompt} (default "prompt")
```

Reply:
curious what @ampinsk thinks. i have zero strong feelings. i'll switch to this and we can always change it.